microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -136,13 +140,17 @@ With this release, you can get the ID of a {microshift-short} cluster. When open
 //[id="microshift-4-16-installation-bug-fixes"]
 //=== Installation
 
-//[discrete]
-//[id="microshift-4-16-networking-bug-fixes"]
-//==== Networking
+[discrete]
+[id="microshift-4-16-networking-bug-fixes"]
+=== Networking
 
-//[discrete]
-//[id="microshift-4-16-support-bug-fixes"]
-//==== Support
+Previously, the {microshift-short} load balancer controller tried to update the IP addresses of every `LoadBalancer` service in the cluster. Some of these services, such as those with a defined `loadBalancerClass`, have their own update procedures for external IPs. This conflicted with the {microshift-short} controller. Now, services that have a `loadBalancerClass` are filtered and IP addresses owned by other load balancer services are ignored by {microshift-short}. (link:https://issues.redhat.com/browse/OCPBUGS-30833[OCPBUGS-30833])
+
+[discrete]
+[id="microshift-4-16-support-bug-fixes"]
+=== Support
+
+Previously, when `microshift-etcd` unexpectedly exited, {microshift-short} tried to restart so that `microshift-etcd` could restart, but there was a lingering unit fragment. Every attempt to restart `microshift-etcd` failed, making the system unusable. The `--collect` flag was added to the `systemd-run` invocation used to start `microshift-etcd`. The addition results in systemd cleaning up the unit fragment even if the unit failed. The system now recovers and restarts. (link:https://issues.redhat.com/browse/OCPBUGS-33588[OCPBUGS-33588])
```

Comment:
Would you want to change "The" to "This"? I was confused by what addition was being referred to here. Just a suggestion though!

I think that systemd should be in the possessive format, as the text refers to "systemd**'s** cleaning up . . . "

Reply:
https://spaces.redhat.com/display/CCSENABLE/Possessives+and+apostrophes (no possesives)

Reply:
Thanks for the link. The info doesn't specifically call out gerunds, but that's okay!